### DISCUSSION TITLE
Is there any workaround for switches support?

### DISCUSSION THREAD

**Discussion by kyraocean** [hass-emulated-hue/core#172] [Q&A] [answered]
I noticed that "switches and other devices" are listed under future plans. Is there any way to enable a switch or group of switches in the meantime? I want to use Emulated Hue with my Harmony Hub to control both lights and plugs (speakers, accent lights) via the remote buttons. 

Thank you for the revised Emulated Hue hub! It's much appreciated.

**Comment by alexyao2015** [comment, marked as answer]
You can create template lights which control switches in home assistant.

**Comment by weissjuergen** [comment]
It seems ther is an easier workaround now. You can create a helper for a switch Type "Change device type of a switch" and create a light that references your switch of whatever integration. It is being recognized in the emulated Hue instantly. 3-4 Mouseclicks that's it.
